packages/react/src/components/Tag/Tag-test.js

```diff
@@ -6,11 +6,12 @@
  */
 
 import { Add } from '@carbon/icons-react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
```

Comment:
```suggestion
import { render, screen } from '@testing-library/react';
```

Doesn't seem like this is being used? If needed consider [userEvent](https://testing-library.com/docs/user-event/intro/) instead.

Reply:
`fireEvent` was not used anywhere ,Good catch ! Thanks !! 🙌